handlers.go

```diff
@@ -679,7 +679,29 @@ func (cm *ContentManager) addDatabaseTrackingToContent(ctx context.Context, cont
 	var objects []*Object
 	cset := cid.NewSet()
 
+	defer func() {
+		cm.inflightCidsLk.Lock()
```

Comment:
we could instead lock and release on each iteration, but this should be a zoomy zoom loop